codalab/apps/web/static/js/bundle/bundle_interface.js

```diff
@@ -155,15 +158,22 @@ var Bundle = React.createClass({
         var editing = this.state.editing;
         var tableClassName = 'table' + (editing ? ' editing' : '');
         var editButtonText = editing ? 'cancel' : 'edit';
-        /// ------------------------------------------------------------------
-        if(editing){
+
+        if (editing)
             saveButton = <button className="btn btn-success btn-sm" onClick={this.saveMetadata}>save</button>
```

Comment:
Yes, should be semicolon.  There are still tons of style violations, which I hope we can fix all at once in another PR.